src/Raven.Server/Documents/Revisions/RevisionsStorage.cs

```diff
@@ -1514,27 +1504,29 @@ private bool PrepareRevertedRevisions(DocumentsOperationContext writeCtx, Parame
                 IEnumerable<Table.TableValueHolder> tvrs = null;
                 if (collections != null)
                 {
-                    foreach(var collection in collections)
+                    tvrs = Enumerable.Empty<Table.TableValueHolder>();
+                    foreach (var collection in collections)
```

Comment:
need to ensure that we don't get a set with `["Users", "users"]`, make sure the comparer is valid.

Reply:
Done